File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution 
{
    public int[] twoSum(int[] nums, int target) 
    {
        int i = 0;
        int j = nums.length - 1;

        while(nums[i] != nums[j])
        {
            int result = nums[i] + nums[j];

            if(result == target)
            {
                return new int[]{i, j};                
            }
            else if(result < target)
            {
                i++;
            }
            else if(result > target)
            {
                j--;
            }
        }

        return null;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        result = []
        hashmap = {}

        for num in nums:
            hashmap[num] = 1 + hashmap.get(num, 0)
        
        for key in hashmap:
            if(hashmap[key] >= k):
                result.append(key)

        return result